src/iuse.cpp

```diff
@@ -8111,9 +8111,12 @@ int iuse::washclothes( player *p, item *it, bool, const tripoint& )
     p->add_msg_if_player( _( "You washed your clothing." ) );
     p->mod_moves( -3000 );
```

Comment:
Long action isn't mandatory, but it should have a limited move cost until then.
We should have some strict limit on non-long action move cost, like 500 or so.